.github/workflows/post-release.yml

```diff
@@ -44,3 +44,5 @@ jobs:
       run: node ./bin/create-docs-pr.js --tag v${{ steps.get_tag.outputs.latest_tag }}
       env:
         GITHUB_TOKEN: ${{ secrets.NODE_AGENT_GH_TOKEN }}
+        GITHUB_USER: ${{ secrets.NODE_AGENT_CI_USER_NAME }}
```

Comment:
can you put the values of these in last pass please?

Reply:
Updated in lastpass.